src/client/components/Carousel/Carousel.component.js

```diff
@@ -25,14 +25,21 @@ const Carousel = ({ imageArray, show }) => {
         &gt;
       </button>
       {imageArray.slice(current, current + show).map((image) => {
-        return <img src={image} alt="product images" className="image" />;
+        return (
+          <img
+            // eslint-disable-next-line
+            src={require(`../../assets/images/${image.split('/')[4]}`)}
+            alt="product images"
```

Comment:
As mentioned Zaki, `alt` has to describe an image.
And it shouldn`t be the same for every image.
Instead of  alt="product images" I think will be better alt={product.name}

Reply:
Thanks, Olena. I have updated the alt part.